pom.xml

```diff
@@ -248,30 +283,17 @@
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-step-api</artifactId>
       <optional>true</optional>
-      <exclusions>
-        <exclusion>
-          <groupId>org.jenkins-ci.plugins</groupId>
-          <artifactId>*</artifactId>
-        </exclusion>
-      </exclusions>
```

Comment:
Unnecessary as of the latest plugin BOM.

Reply:
Are all instances of this unnecessary, i.e. [this one as well](https://github.com/jenkinsci/jira-plugin/blob/master/pom.xml#L274)?

Reply:
Not sure offhand.